bonsai/examples/governance/methods/README.md

```diff
@@ -0,0 +1,34 @@
+## zkVM Methods
+
+This directory contains the [zkVM] portion of your [Bonsai] application. 
+This is where you will define one or more [guest programs] to act as a coprocessor to your [on-chain logic].
+
+> In typical use cases, the only code in this directory that you will need to edit is inside [`guest/src/bin`]. 
```

Comment:
This makes it sound like the user needs to modify code in `guest/src/bin` to use the governance example. I don't think that's true (but maybe I'm confused?)